scss/styles.scss

```diff
@@ -141,3 +121,65 @@
   padding: 10px;
   text-transform: uppercase;
 }
+
+.front-grid {
+  display: grid;
+  grid-template-columns: repeat(3, max-content);
+  grid-template-areas:
+    'header header header'
+    'react angular vue'
+    'react elm polymer'
+}
+
+.tech.react  {
```

Comment:
no need to add the `.tech` to any of this since the properties here don't override the ones in `.tech`